Packs/CortexXDR/Playbooks/Cortex_XDR_-_T1036_-_Masquerading.yml

```diff
@@ -0,0 +1,1045 @@
+        "position": {
+          "x": 1190,
+          "y": 2500
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "122":
+    id: "122"
+    taskid: d2213fdb-3690-48c4-820a-7f8a0d826f1f
+    type: condition
+    task:
+      id: d2213fdb-3690-48c4-820a-7f8a0d826f1f
+      version: -1
+      name: Execution was Blocked?
+      description: ""
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "82"
+      "no":
+      - "123"
+    separatecontext: false
+    conditions:
+    - label: "no"
+      condition:
+      - - operator: notContainsString
+          left:
+            value:
+              complex:
+                root: PaloAltoNetworksXDR.Incident.alerts
+                filters:
+                - - operator: in
+                    left:
+                      value:
+                        simple: PaloAltoNetworksXDR.Incident.alerts.alert_id
+                      iscontext: true
+                    right:
+                      value:
+                        simple: inputs.AlertID
+                      iscontext: true
+                    ignorecase: true
+                accessor: action
+            iscontext: true
+          right:
+            value:
+              simple: Blocked
+          ignorecase: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: PaloAltoNetworksXDR.Incident.alerts
+                filters:
+                - - operator: in
+                    left:
+                      value:
+                        simple: PaloAltoNetworksXDR.Incident.alerts.alert_id
+                      iscontext: true
+                    right:
+                      value:
+                        simple: inputs.AlertID
+                      iscontext: true
+                    ignorecase: true
+                accessor: actor_process_os_pid
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1180,
+          "y": 560
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "123":
+    id: "123"
+    taskid: e34a609e-4e8c-477e-8541-ce0f1419412c
+    type: title
+    task:
+      id: e34a609e-4e8c-477e-8541-ce0f1419412c
+      version: -1
+      name: Early Containment
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "124"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1540,
+          "y": 730
+        }
+      }
+    note: false
+    timertriggers:
+    - fieldname: containmentsla
+      action: start
```

Comment:
if you start a timer, you should also stop it.
if you already added timers, why not adding the rest of them?